### PR TITLE
Add Phytron VSS 80.200.5

### PR DESCRIPTION
**Pull request by cdn-lnk**
[Motor manual](https://github.com/paulscherrerinstitute/ecmccfg/files/12599269/ds-vacuum-en.pdf)

### CODE REVIEW

**Comment by anderssandstrom** [comment]
Wrong motor type description in headers: VSS-52.200.2.5 but should be VSS-80.200.5

File: hardware/Motors/ecmcEL7037-Motor-Phytron-VSS-80.200.5.cmd [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#-d /**
+#-d   \brief hardware script for EL7037-Motor-Phytron-VSS-52.200.2.5
```
Comment (kivel) [inline hardware/Motors/ecmcEL7037-Motor-Phytron-VSS-80.200.5.cmd:2]: VSS-80.200.5

File: hardware/Motors/ecmcEL7037-Motor-Phytron-VSS-80.200.5.cmd [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+#-d   \details Parametrization of EL7037 for motor Phytron VSS 52.200.2.5
```
Comment (kivel) [inline hardware/Motors/ecmcEL7037-Motor-Phytron-VSS-80.200.5.cmd:3]: VSS-80.200.5

File: hardware/Motors/ecmcEL7041-Motor-Phytron-VSS-80.200.5.cmd [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#-d /**
+#-d   \brief hardware script for EL7041-Motor-Phytron-VSS-52.200.2.5
```
Comment (kivel) [inline hardware/Motors/ecmcEL7041-Motor-Phytron-VSS-80.200.5.cmd:2]: VSS-80.200.5

File: hardware/Motors/ecmcEL7041-Motor-Phytron-VSS-80.200.5.cmd [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+#-d   \details Parametrization of EL7041 for motor Phytron VSS 52.200.2.5
```
Comment (kivel) [inline hardware/Motors/ecmcEL7041-Motor-Phytron-VSS-80.200.5.cmd:3]: VSS-80.200.5

File: hardware/Motors/ecmcEL7047-Motor-Phytron-VSS-80.200.5.cmd [hunk 5]
```diff
@@ -0,0 +1,14 @@
+#-d /**
+#-d   \brief hardware script for EL7047-Motor-Phytron-VSS-52.200.2.5
```
Comment (kivel) [inline hardware/Motors/ecmcEL7047-Motor-Phytron-VSS-80.200.5.cmd:2]: VSS-80.200.5

File: hardware/Motors/ecmcEL7047-Motor-Phytron-VSS-80.200.5.cmd [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+#-d   \details Parametrization of EL7047 for motor Phytron VSS 52.200.2.5
```
Comment (kivel) [inline hardware/Motors/ecmcEL7047-Motor-Phytron-VSS-80.200.5.cmd:3]: VSS-80.200.5

**Review by kivel — CHANGES_REQUESTED** [review]
as per comment from @anderssandstrom

**Comment by cdn-lnk** [comment]
Hi @anderssandstrom and @kivel ,

Guess I was having a bad day when I pressed the push button. 😅
It should be all fixed now, thanks for catching that.

**Comment by kivel** [comment]
@cdn-lnk no worries, that's what reviews are for.

**Pull request merged**